### ISSUE TITLE
allow multiple values for `kb_options`

### ISSUE THREAD

**Issue by mmahmoudian** [hyprwm/Hyprland#7513]
### Description

Two disclaimers first:
1. I searched through the open and closed issues using "kb_options" and at least based on title I didn't find anything that is even remotely related.
2. I'm not sure if this is a bug or currently the intentional behavior, but based on the documentation it seems I have to file feature request rather than bug report

-------

I like many would like to two do two things with my keyboard:
1. using both shifts to toggle keyboard layout
2. using Capslock as a Ctrl (Emacs folks do this, and Vim folks replace it with Esc)

So I need both `kb_options = ctrl:nocaps` and `kb_options = grp:shifts_toggle` at the same time. Based on the [Documentation](https://wiki.hyprland.org/Configuring/Variables/#input), the `kb_options` should be a string, similar to `kb_layout`. So I logically went for the comma-separated approach, exactly siumilar to `kb_layout`. Also the name id optionS which implies more than one:

```
input {
    kb_layout = fi,ir
    kb_options = ctrl:nocaps,grp:shifts_toggle
    ....
}
```

But this didn't work. So I though I define multiple of them similar to `animation` and `bind`:

```
input {
    kb_layout = fi,ir
    kb_options = ctrl:nocaps         # this is to convert CapsLock to Ctrl
    kb_options = grp:shifts_toggle   # this toggles keyboard layout to change languages 
    ....
}
```

But this didn't work either because only the last one gets into affect (overwrite the previous ones).

I believe the cleanest approach is to have this feature as comma-separated. Although, ideally I believe both forms of definition should be allowed. Basically the `kb_options` should have additive/cumulative approach rather than over-writing.

Thanks for the great and beautiful software.

**Comment by MahouShoujoMivutilde** [comment]
Its supports multiple values, and your setup works for me - layout is switched on both shifts, and caps is ctrl.

```
input {
    kb_layout = us,ru # fi,ir too works
    # kb_options = caps:escape,grp:toggle
    kb_options = ctrl:nocaps,grp:shifts_toggle
}
```

Comma is the correct way, yes.

This is on 66586c38f53f16bcf762f019359a3c9042546a72, but I've been running 

```
device {
    name = logitech-usb-keyboard

    kb_layout = us,ru
    kb_options = caps:escape,grp:toggle
}
```

for more than a year at this point.

There was an option to disable config autoreload. Maybe you just have that and so it doesn't load new configuration? You can always just `hyprctl reload`.

**Comment by mmahmoudian** [comment]
You are right 😳 now it works. I had this bug report in my todo list to be reported and didn't check all the conditions when I posted, but also I didn't see anything in the release notes that this was fixed or something. 🤔

I'm confident this was an issue because I ask other hyprland users on Mastodon if they know how to approach this and no one had a solution.

Thanks for your nice comment. I'll close this issue now
